devops/linux/docker/deploy-local.sh

```diff
@@ -0,0 +1,162 @@
+# #!/bin/bash
+# Set JEMPI_HOME environment variable
+cd ../../..
+export JEMPI_HOME=$(pwd)
+export JAVA_VERSION=17.0.8.1-tem
+echo "Setting JEMPI_HOME to: $JEMPI_HOME"
+#JEMPI_CONFIGURATION_PATH=$JEMPI_HOME/JeMPI_Apps/JeMPI_Configuration/ethiopia/config-ethiopia-3.json
```

Comment:
Can we remove this commented out code

Reply:
@walisc 
Yes sure.
Still I'm working on this.
this is still not ready for review

Reply:
Oh, sorry @sushantpatil1214 . I got confused with your message in slack. I thought you want someone to review this PR. Noted! 👍🏽 

p.s there is an option to convert it to draft if you want to use that (under the reviewers section) and then only open it once ready.... but up to you 👍🏽 

Thanks @sushantpatil1214 !